packages/packages/v1-adapters/src/dispatchers/dispatchers.ts

```diff
@@ -37,8 +37,14 @@ export function openRoute( route: string ): Promise<void> {
 	}
 }
 
-export function isRouteActive( route: string ): boolean {
+export function isRouteActive( route: string ) {
 	const extendedWindow = window as unknown as ExtendedWindow;
 
 	return !! extendedWindow.$e?.routes?.isPartOf( route );
 }
+
+export function getCurrentEditMode() {
+	const extendedWindow = window as unknown as ExtendedWindow;
+
+	return extendedWindow.elementor.channels.dataEditMode.request( 'activeMode' );
+}
```

Comment:
Why those 2 related to dispatchers?

Reply:
actually they're not really related, just didn't know where to put them, so I counted on the fact that you'll catch that in the review 😂